src/components/Radio/RadioIndex.tsx

```diff
@@ -0,0 +1,19 @@
+import { label } from "aws-amplify"
```

Comment:
Don't know how this got past the linter, but we should remove imports we are not using